src/gui/guiFormSpecMenu.cpp

```diff
@@ -868,12 +947,26 @@ void GUIFormSpecMenu::parseDropDown(parserData* data, const std::string &element
 
 		MY_CHECKPOS("dropdown",0);
 
-		v2s32 pos = getElementBasePos(false, &v_pos);
+		v2s32 pos;
+		v2s32 geom;
+		core::rect<s32> rect;
 
-		s32 width = stof(parts[1]) * spacing.Y;
+		if (data->real_coordinates) {
 
-		core::rect<s32> rect = core::rect<s32>(pos.X, pos.Y,
-				pos.X + width, pos.Y + (m_btn_height * 2));
+			std::vector<std::string> v_geom = split(parts[1],',');
+			MY_CHECKGEOM("dropdown",1);
+
+			pos = getRealCoordinateBasePos(false, &v_pos);
+			geom = getRealCoordinateGeometry(&v_geom);
+			rect = core::rect<s32>(pos.X, pos.Y, pos.X+geom.X, pos.Y+geom.Y);
```

Comment:
Using dropdowns with only width set is impossible with `real_coordinates` change or document this behaviour.

Reply:
You shouldn't be allowed to not set the height.  All elements have explicitly set heights with `real_coordinates` because otherwise dropdowns will have a weird height that doesn't match up with everything else.

I'll document this.